vms/platformvm/warp/message/codec.go

```diff
@@ -23,7 +23,6 @@ func init() {
 		lc.RegisterType(&SubnetConversion{}),
 		lc.RegisterType(&RegisterSubnetValidator{}),
 		lc.RegisterType(&SubnetValidatorRegistration{}),
-		lc.RegisterType(&SetSubnetValidatorWeight{}),
 		lc.RegisterType(&SubnetValidatorWeight{}),
```

Comment:
Small inconsistency, in the ACP I had kept the message as `SetSubnetValidatorWeightMessage` rather than `SubnetValidatorWeightMessage`. 

Fine with going with whichever though

Reply:
I don't see the name `SetSubnetValidatorWeightMessage` anywhere in the ACP 🤔

Reply:
Decided to stay with `SubnetValidatorWeightMessage` and will update the spec to name it as such